docs/content/RadioButton.md

```diff
@@ -0,0 +1,92 @@
+---
+title: RadioButton
+description: Use radio buttons when users need to select one option from a list
+status: Alpha
+source: https://github.com/primer/react/blob/main/src/RadioButton.tsx
+storybook: '/react/storybook?path=/story/forms-radio-button--default'
+---
+
+import {ComponentChecklist} from '../src/component-checklist'
+
+## Default example
+
+```jsx live
+<>
+  <RadioButton value="active radio button value" name="radio-group-name" />
+  <RadioButton value="active radio button value" name="radio-group-name" />
+  <RadioButton disabled value="inactive radio button value" name="radio-group-name" />
+</>
+```
+
+<Note>
+Please use a <a href="/react/Checkbox">Checkbox</a> if the user needs to select more than one option in a list
+</Note>
```

Comment:
Weird MDX quirk: If you add newlines, you can use markdown syntax within component children:
```suggestion
<Note>

Please use a [Checkbox](/Checkbox) if the user needs to select more than one option in a list.

</Note>
```

Reply:
Wow, I had no idea! This is great.